### DISCUSSION TITLE
Getting Started

### DISCUSSION THREAD

**Discussion by 0pindar0** [Homebrew/discussions#4852] [Getting started]
### Output of `brew config`

```shell
?
```

### Output of `brew doctor`

```shell
?
```

### Description of issue

I downloaded home-brew onto Mac. I can't find it anywhere. I was looking for some help on where to look and how to configure it to UTM. seems straightforward enough, but the few bumps in the road are throwing me off. Any help is great. Thank you.

**Comment by SMillerDev** [comment]
How did you download it?
